apps/api/src/app/order/order.service.ts

```diff
@@ -395,6 +397,26 @@ export class OrderService {
       };
     }
 
+    if (filtersByDataSource?.length > 0) {
+      where.SymbolProfile = {
+        dataSource: {
+          in: <DataSource[]>filtersByDataSource.map(({ id }) => {
+            return id;
+          })
+        }
+      };
+    }
+
+    if (filtersBySymbol?.length > 0) {
+      where.SymbolProfile = {
```

Comment:
We will (probably) always use `dataSource` and `symbol` filters together. I don't think, this works as it would override `where.SymbolProfile`.

Can you do something like this?

```
if (filterByDataSource && filterBySymbol) {
  if (where.SymbolProfile) {
    // Extend where.SymbolProfile
  } else {
    where.SymbolProfile = {
      AND: [
        { dataSource: filterByDataSource },
        { symbol: filterBySymbol }
      ]
    }
  }
}
```